src/sentry/static/sentry/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -107,8 +107,11 @@ export function getOnboardingTasks(
         // instructions for the corresponding project to prompt the user to send an event
         // to the new project.
         const projectSlug =
-          project && org.projects
-            ? org.projects.find(p => p.id === `${project}`)!.slug
+          project &&
+          org.projects &&
+          org.projects.find(p => p.id === `${project}`) &&
+          org.projects.find(p => p.id === `${project}`).slug
+            ? org.projects.find(p => p.id === `${project}`).slug
```

Comment:
Can we split this up? We're calling `find()` on `org.projects` (which can have thousands of items) multiple times.

Reply:
Alternatively, you could use the optional chaining operator:

```
 project && org?.projects.find(p => p.id === `${project}`)?.slug
```

Reply:
Changed it. It's now calling `find` twice - lmk if you think I should go ahead and extract the call into a variable.